src/js/menu/media-captions-menu-button.ts

```diff
@@ -104,7 +104,9 @@ class MediaCaptionsMenuButton extends MediaChromeMenuButton {
    */
   get invokeTargetElement(): HTMLElement | null {
     if (this.invokeTarget != undefined) return super.invokeTargetElement;
-    return getMediaController(this)?.querySelector('media-captions-menu');
+    const controller = getMediaController(this);
+    if (!controller) return null;
+    return controller.querySelector('media-captions-menu') as HTMLElement | null;
```

Comment:
This could be done the same way as it is for the `media-rendition-menu-button`
```suggestion
    return controller ? controller.querySelector('media-captions-menu') : null;
```